fili-core/src/main/java/com/yahoo/bard/webservice/web/ErrorMessageFormat.java

```diff
@@ -73,6 +73,8 @@
     SORT_METRICS_NOT_SORTABLE_FORMAT("Sorting not possible on metric(s) '%s'."),
     SORT_METRICS_UNDEFINED("Metric(s) in sort expression '%s' do not exist."),
 
+    DATE_TIME_SORT_VALUE_INVALID("dateTime must always be the first field in the sort list"),
```

Comment:
This error message is displayed only when `dateTime` is part of the sort list but not the first one. If there is no sort list or if there is no request for `dateTime` sort, we will not reach till this error.